cloud/amazon/cloudformation.py

```diff
@@ -14,12 +14,22 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+# upcoming features:
+# - AWS-native YAML support
+# - Ted's multifile YAML concatenation
+# - changesets (and blocking/waiting for them)
```

Comment:
Would you want to chat on IRC or expand in a "feature_idea" issue on how these would be used? Mostly I see changesets as an interactive thing where the operator can say "Wait wait, that's not what I meant" before applying.